eZ/Bundle/EzPublishCoreBundle/Command/RegenerateUrlAliasesCommand.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Command;
```

Comment:
can we add `declare(strict_types=1);`?

Reply:
Fixed via f96dd03 along with some other fixes.